.kitchen.yml

```diff
@@ -1,16 +1,42 @@
 ---
 driver:
-  name: vagrant
+  name: docker
 
 provisioner:
   name: chef_solo
+  require_chef_omnibus: 11.16.4
+  solo.rb:
+    environment: aws-us-west-2-staging
 
 platforms:
-  - name: ubuntu-12.04
+  - name: ubuntu-14.04
+    driver:
+      image_id: ami-3be3f20b
```

Comment:
For attributes like this that are specific to the person/group running the build, you can take advantage of the fact that Test Kitchen runs its config through an ERB parser first. So you can do things like:

```
driver:
  image_id: <%= ENV['AWS_IMAGE_ID'] %>
```

and then feed them in as environment variables, whether set locally on a dev machine or set on a CI server as part of a build config.